python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -367,6 +382,8 @@ def _is_supported(arg, supported: set):
             for col in arg:
                 if isinstance(arg[col], list):
                     _global_set = _global_set.union(set(arg[col]))
+                elif isinstance(arg[col], dict):
```

Comment:
Does order matter for `_global_set`? If it does, using `set` can sometimes change the order and give unexpected results.

Reply:
Shouldn't matter here, since we only need `_global_set` to check if our aggs are a subset of `supported`. Ordering is more of a concern with `_redirect_aggs()` since that function returns a copy of the aggs that's used for the remainder of the groupby. AFAIK that should be good

Reply:
Great, sounds good! For the most part the code looks pretty good to me!